Tests/MapboxMapsTests/Location/Puck/Puck2DTests.swift

```diff
@@ -225,8 +225,7 @@ final class Puck2DTests: XCTestCase {
 
         puck2D.isActive = true
 
-        var expectedLayer = makeExpectedLayer()
-        expectedLayer.shadowImage = nil
+        let expectedLayer = makeExpectedLayer()
```

Comment:
Let's update this test to be named testActivatingPuckWithNilImages. At the beginning of the test, nil out all 3 images, then on these lines, nil out expectedLayer.bearingImage.